qemu/tests/cfg/block_stream.cfg

```diff
@@ -108,3 +121,36 @@
                     when_streaming = "pause_job set_speed resume_job cancel start wait_for_finished"
                     Host_RHEL.m6:
                         when_streaming = "set_speed cancel start wait_for_finished"
+        - data_integrity:
```

Comment:
I'd suggest to append the prefix "with_" into these variant names to avoid naming conflict.